test/apps/iodemo/io_demo.cc

```diff
@@ -1610,14 +1612,11 @@ class DemoClient : public P2pDemoCommon {
 
             elapsed_time = curr_time - start_time;
             if (elapsed_time > _test_opts.client_timeout) {
-                std::stringstream ss, list_uncomp_ss;
-                size_t uncomp_to_disconnect =
-                        get_uncompleted_servers_info(list_uncomp_ss);
+                std::stringstream ss;
```

Comment:
why need intemediate stringstream buffer here and at line 1560? Maybe use UcxLog directly?

Reply:
we collecting printing information to print it as a single trace

Reply:
just wanted to comment the same thing: get_server_info_uncompleted_ops needs ss. get_uncompleted_servers_info does not.

Reply:
UcxLog dumps its own buffer to stdout in dtor

Reply:
done